bindings/core/src/method_handler/wallet.rs

```diff
@@ -297,12 +297,11 @@ pub(crate) async fn call_wallet_method_internal(wallet: &Wallet, method: WalletM
             Response::PreparedTransaction(PreparedTransactionDataDto::from(&data))
         }
         WalletMethod::PrepareTransaction { outputs, options } => {
-            let token_supply = wallet.client().get_token_supply().await?;
             let data = wallet
                 .prepare_transaction(
                     outputs
                         .into_iter()
-                        .map(|o| Ok(Output::try_from_dto_with_params(o, token_supply)?))
+                        .map(|o| Ok(Output::try_from(o)?))
```

Comment:
clippy gonna complain

Reply:
Why? What can I do because of the Ok ?

Reply:
pretty sure this can be reduced to ```.map(Output::try_from)```

Reply:
No, we need to `?` to convert the error

Reply:
At least for this one, I'll try the others

Reply:
That can happen in the collect instead since they are all the same error

Reply:
There was one

Reply:
No they are not the same error

Reply:
No, I mean all the collected results have the same error

Reply:
Ok so maybe I'm not understanding what you mean. Also in a hurry at the airport :trollface: 
Can you show me because simply `.map(Output::try_from)` doesn't work.

Reply:
```
.map(Output::try_from)
      .collect::<Result<Vec<Output>, _>>()?,
```

I removed the `Result` import and manually used `crate::Result` in the return type, but that's all

Reply:
I see, thanks, done

Reply:
If it was only me I would remove these Result typedefs, I hate them